src/reference/build_cellranger_arc_reference/config.vsh.yaml

```diff
@@ -0,0 +1,65 @@
+functionality:
+  name: build_cellranger_arc_reference
+  namespace: reference
+  description: Build a Cell Ranger-arc and -atac compatible reference folder from user-supplied genome FASTA and gene GTF files. Creates a new folder named after the genome.
+  authors:
+    - __merge__: /src/authors/vladimir_shitov.yaml
+      roles: [ author ]
+  arguments:
+    # inputs
+    - type: file
+      name: --genome_fasta
+      required: true
+      description: Reference genome fasta.
+      example: genome_sequence.fa.gz
+    - type: file
+      name: --annotation_gtf
+      required: true
+      description: Reference annotation.
+      example: annotation.gtf.gz
+    - type: file
+      name: --motifs_file
+      required: false
+      description: Transcription factor motifs in JASPAR format. See https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/advanced/references
+      example: JASPAR2024_CORE_non-redundant_pfms_jaspar.txt.modified
+    - type: string
```

Comment:
I think this one is also `multiple: true`?

Reply:
In practice, it should be only chrM, at least for human data. But you are right, it should be an array. I modified the code